dbos/system_database.py

```diff
@@ -1054,14 +1094,42 @@ def start_queued_workflows(
                     )
                     .values(status=WorkflowStatusString.PENDING.value)
                 )
-                if result.rowcount > 0:
-                    ret_ids.append(id)
+
+                # Then give it a start time
+                c.execute(
+                    SystemSchema.job_queue.update()
+                    .where(SystemSchema.job_queue.c.workflow_uuid == id)
+                    .values(started_at_epoch_ms=start_time_ms)
```

Comment:
Probably compute start_time_ms = int(time.time() * 1000) again. Otherwise, they all have the same start_time_ms...